api/app/pytest.ini

```diff
@@ -1,7 +1,7 @@
 [pytest]
 filterwarnings =
     ignore::DeprecationWarning
-log_format = %(asctime)s %(levelname)s %(message)s
+log_format = %(levelname)s %(message)s
 log_date_format = %Y-%m-%d %H:%M:%S
-; log_cli=true
-; log_level=DEBUG
+log_cli=true
```

Comment:
I'd prefer we don't commit these - it makes for an extremely verbose log file. Was this a mistake or do you really need more detailed logging?

Reply:
Hmm... I set these because I wanted to print out things to debug while testing. Is there another way to enable logging when running tests?

also, quick question, do we store log files for testing?

Reply:
What I've been doing is setting it to log_cli=true and log_level=DEBUG when I'm trying to fix tests or troubleshoot - but when I just want to check if my tests are passing, having a crazy verbose list of results is just too much.
The logs are around for a while in the github actions.

Reply:
Okay, changes have been reverted!

Reply:
Doesn't log_cli=true also give a lot of additional stuff?